Plugin is not implemented on v7.0.0 scenarigo binary in releases page

## Overview

Thanks for awesome tool! 😃 
I tried using scenarigo but plugin seems not working well.

## Details

I created a minimum example to reproduce this.
https://github.com/konifar/scenarigo-plugin-example

The error message is below.

```
$ ./v0.7.0_Darwin_x86_64/scenarigo run
$ --- FAIL: github.yml (0.00s)
$     --- FAIL: github.yml/get_scenarigo_repository (0.00s)
$             failed to open plugin: plugin: not implemented
```

Plugin has been introduced since Go 1.8 and scenarigo supports Go 1.16 or later, so it should be no problem.

## References

Go binary needs cgo to use plugin?
[golang/go#19569 plugin: requires CGO_ENABLED=1](https://github.com/golang/go/issues/19569)

I tried removing `CGO_ENABLED=0` from [.goreleaser.yml#11](https://github.com/zoncoen/scenarigo/blob/0fbf7cc825a57f61d4495644b39242783e542383/.goreleaser.yml#L11) and used a binary which build on local, it worked well. (I'm not sure it should be removed actually...)

Probably this issue is also related.
https://github.com/kubernetes-sigs/kustomize/issues/2299

## Environments

- scenarigo [v0.7.0_Darwin_x86_64](https://github.com/zoncoen/scenarigo/releases/tag/v0.7.0)
- macOS Catalina v10.15.7

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -8,7 +8,7 @@ builds:
   - -X github.com/zoncoen/scenarigo/version.version={{.Version}}
   - -X github.com/zoncoen/scenarigo/version.revision=
   env:
-  - CGO_ENABLED=0
+  - CGO_ENABLED=1 # for plugin package
 archives:
 - name_template: "{{ .ProjectName  }}_v{{ .Version }}_{{ .Os  }}_{{ .Arch  }}"
   replacements:

